core/common/src/MaterialProps.ts

```diff
@@ -63,14 +63,14 @@ export interface TextureMapProps {
   /** Weight at which to combine diffuse image and color; if undefined, defaults to 1.0 */
   pattern_weight?: number;
   /** If true, override the mapping mode with constant LOD mapping for the normal map, defaults to false. */
-  pattern_useConstantLod?: boolean;
-  /** The number of times the texture is repeated if pattern_useConstantLod is true.  Increasing this will make the texture pattern appear smaller, decreasing it will make it larger. Defaults to 1.*/
+  pattern_useconstantlod?: boolean;
```

Comment:
The camel-cased one is `@public` which means you can't delete it without breaking people's code, which we can't do in a minor release.
Mark it `@deprecated in 4.4. It never functioned properly - use [[pattern_useconstantlod]] instead`.

Reply:
done.